src/event-distribution/events/message.ts

```diff
@@ -42,7 +42,7 @@ export const extractMessageInfo: ExtractInfoForEventFunction<
   DiscordEvent.MESSAGE
 > = (message) => {
   const split = message.content.split(" ");
```

Comment:
```suggestion
  const split = message.content.toLowerCase().split(" ");
  const trigger = split[0];
  const subTrigger = split[1];
```
The way you've done is basically having it 50/50, any command send the trigger is lowerCased but not the subtrigger, for example `!group Join` would not work.